src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -113,6 +113,7 @@ Referenced assemblies:
 - 'PresentationCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'SonarAnalyzer, Version=8.26.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+- 'SonarLint.VisualStudio.CFamily, Version=4.38.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
```

Comment:
There was a project reference to the CFamily project before, but I guess the code is now actually using something from the assembly.